backend/apps/api/serializers/user/profile/overview.py

```diff
@@ -0,0 +1,21 @@
+import copy
+
+from rest_framework import serializers
+
+from ....serializers.comment import CommentDetailSerializer
+from ....serializers.post import PostSerializer
```

Comment:
lets absolute imports them since they are different app serializer instead of this explicit relative imports.
```suggestion
from apps.api.serializers.comment import CommentDetailSerializer
from apps.api.serializers.post import PostSerializer
```

Reply:
> lets absolute imports them since they are different app serializer instead of this explicit relative imports.

@moonlitgrace, yeah i also thought of using absolute imports but as i could see other relative imports in the code base got confused so kept those above imports as relative imports, now as u confirmed i will now use absolute imports 👍